### PR TITLE
MISUV-2810 Create Repayment connector

### PR DESCRIPTION
**Pull request by andriikondiuk**

### CODE REVIEW

**Review by chandraAndem — CHANGES_REQUESTED** [review]
Very minor changes required.

File: conf/application.conf [hunk 1]
```diff
@@ -232,6 +237,10 @@ mtd-subscription-service {
   url = "https://www.gov.uk/guidance/use-software-to-send-income-tax-updates"
 }
 
+repayment-api {
```
[the remark this answers is not in the document]
Comment (andriikondiuk) [reply conf/application.conf]: Thanks, It's been removed. Done

File: app/connectors/RepaymentConnector.scala [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import play.api.http.Status.{ACCEPTED, INTERNAL_SERVER_ERROR}
+import play.api.libs.json.Json
+import uk.gov.hmrc.http.HttpReads.Implicits.readRaw
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class RepaymentConnector @Inject()(val http: HttpClient,
+                                   val auditingService: AuditingService,
```
[the remark this answers is not in the document]
Comment (andriikondiuk) [reply app/connectors/RepaymentConnector.scala]: Thanks, I've just removed AuditingService from connector and from the test. Done

File: conf/application.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (andriikondiuk) [reply conf/application.conf]: Thanks, It's been removed. Done

**Review by chandraAndem — APPROVED** [review]

**Pull request merged**